oidc-exchange.py

```diff
@@ -161,13 +191,15 @@ def assert_successful_audience_call(resp: requests.Response, domain: str):
             )
 
 
-def render_claims(token: str) -> str:
+def extract_claims(token: str) -> dict[str, typing.Any]:
```

Comment:
Would `object` work here?
```suggestion
def extract_claims(token: str) -> dict[str, object]:
```

I learned recently that MyPy docs recommend this instead of `Any`, whenever possible.

Reply:
I think so! TIL they recommend that, I've been using `Any` for years.

Reply:
I've been exposed to more MyPy things this year, between observing what the pytest and aiohttp co-maintainers do, and experimenting a lot with MyPy coverage reports which I didn't know were a thing a few years ago — I even started uploading those to Codecov and will hopefully bring that practice here.

@woodruffw could you also apply `object` to other places that I didn't explicitly mark?